api/apps/api/src/modules/features/features/domain/feature.ts

```diff
@@ -0,0 +1,40 @@
+import { FeatureFragment } from './feature-fragment';
+import { CustomFeature } from './custom-feature';
+
+export enum FeatureTag {
+  Species = 'species',
+  Bioregional = 'bioregional',
+}
+
+export class Feature {
+  constructor(
+    private readonly id: string,
+    private readonly tag: FeatureTag,
+    private readonly fragments: FeatureFragment[],
+    private readonly name: string,
+  ) {}
+
+  split(byKey: string): CustomFeature[] {
```

Comment:
we should also (now or in a second step, but as part of this epic) support non-exhaustive splits, and these will actually be very common with the current bioregional "über-feature", where we users may only want to even select one bioregion kind (so, splitting the über-feature by bioregion kind may give us 30 subsets mapping to different kinds of deserts, savannas, etc. but the user may only be interested in one or two of these, and most likely in a subset in any case)

Reply:
Consumer, to be able to select one of many, should first know the options, right? So, Feature domain cares only about how combining layers work (i.e. contains all important rules). What will user do later on with available pool is another kind of story (and could be rather a simple CRUD). Thoughts?